### PR TITLE
[SIE-102] Create Experiments page

### PR DESCRIPTION
**Pull request by fernandowinfield**
Put together the Experiments page. **This pull requests includes**:
- Creating the `ExperimentsPage` component per design request.
- Extending the `AddExperiment` and `Modal` components to be able to customize the modal's button that opens the form to create a new experiment.
- Modifying the `Modal` styling to prevent overlapping with other content in the page.
- Fixed a small bug on the `Slider` component to prevent it from crashing when no children are given to the slider.
- Left TODOs in `ExperimentsPage` to address in next sprint. These are mostly small adjustments that have to do with solidifying the fields that the 'Experiment' data model must have.

**Missing**: unit tests. I will add these in a separate pull request or here if need be, but wanted to post this pull request in case it's needed for the demo.

Jira task: https://cs6510.atlassian.net/browse/SIE-102

### CODE REVIEW

File: src/stories/pages/ExperimentsPage/ExperimentsPage.js
Comment (alecherryy) [inline src/stories/pages/ExperimentsPage/ExperimentsPage.js:141]: Small thing I noticed, the button gets distorted on hover. You might want to override/adjust the styles accordingly.

File: src/stories/pages/ExperimentsPage/ExperimentsPage.js
Comment (alecherryy) [inline src/stories/pages/ExperimentsPage/ExperimentsPage.js:136]: What is the need for this lone `<div>`?

File: src/stories/pages/ExperimentsPage/ExperimentsPage.js
Comment (alecherryy) [inline src/stories/pages/ExperimentsPage/ExperimentsPage.js:139]: You don't need to use uppercase text, `<h6>` are already all uppercase.

File: src/stories/pages/ExperimentsPage/ExperimentsPage.stories.js
Comment (alecherryy) [inline src/stories/pages/ExperimentsPage/ExperimentsPage.stories.js:17]: **Nitpick Alert**: no need for this line if there are no arguments.

File: src/stories/pages/ExperimentsPage/styles.scss
Comment (alecherryy) [inline src/stories/pages/ExperimentsPage/styles.scss:13]: Ideally pages shouldn't need their own stylesheet. You can apply the margins directly to the Slider component. Also, you can change the `.slider__controls` so it's not floating and add `text-align: right`. This way you don't have to use margin and padding to create more space between sections.

**Review by alecherryy — CHANGES_REQUESTED** [review]
@fernandowinfield I left a few comments, most of them are more suggestions for cleaner code than actual issues. I'll leave it to you to decide what to update.

**Comment by fernandowinfield** [comment]
Thanks, @alecherryy for the awesome suggestions and for executing them. The Experiments page looks much better! I think this pull request is ready to be merged once approved 👍

**Review by cmutuc25 — APPROVED** [review]

**Pull request merged**
